app/models/Osco/warrant.rb

```diff
@@ -0,0 +1,3 @@
+class Osco::Warrant < ApplicationRecord
```

Comment:
We will need to rename this next time we work on it

Reply:
understood